docs/en/guides/How-to-release.md

```diff
@@ -251,3 +251,11 @@ make docker.push
 Once the latest release has been published, you should clean up the old releases from the mirror system.
 1. Update the download links (source, dist, asc, and sha512) on the website to the archive repo (https://archive.apache.org/dist/skywalking).
 2. Remove previous releases from https://dist.apache.org/repos/dist/release/skywalking/.
+
+## Update the Quick Start Versions
+
+We hosted the [SkyWalking Quick Start script](https://skywalking.apache.org/docs/main/latest/en/setup/backend/backend-docker/#start-the-storage-oap-and-booster-ui-with-docker-compose), 
+which is a shell script that helps users to download and start SkyWalking quickly.
+The versions of OAP and BanyanDB are hard-coded in the script, so you need to update the versions in the script.
```

Comment:
This statement is slightly wrong. The versions are not hardcoded, the ones there are default values and can be modified by simply setting the corresponding env var. if we aim to always use the latest versions we can simply set those to `latest` since we have the `latest` docker image tags for both images.

Reply:
I don't think `hardcoded` is a negative statement, and we don't plan to use `latest` automatically. As BanyanDB is still in evolution, we can't guarantee compatibility. 

`hardcoded` aims to remind the release manager(end users don't read this) that, he/she needs to set up the versions matched.